canonical-web-and-design/ubuntu.com
# Fix eslint errors for form validation js

## Done

Fixed linting errors with static/js/src/formValidation.js and moved dependent inline JS to same file

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run `./run exec ./node_modules/.bin/eslint static/js/src/formValidation.js
- Check that there are no eslint errors/warnings
- Visit http://localhost:8001/whitepapers/beta/robotics
- Fill out the form and check that you can see the hidden content
- Visit http://localhost:8001/whitepapers/beta/robotics
- Fill out the form and check that you can see the hidden content


## Issue / Card

Fixes #6885
